src/lib/hooks/use-mobile-nav.tsx

```diff
@@ -0,0 +1,24 @@
+const useMobileNav = () => {
+  const navList: NavLink[] = [
+    {
+      url: "/",
+      text: "Home"
+    },
+    {
+      url: "/newsletter",
+      text: "Newsletter"
+    },
+    {
+      url: "/about",
+      text: "About"
+    },
+    {
+      url: "/contact",
+      text: "Contact Us"
+    }
+  ];
+
+  return navList;
+};
+
+export default useMobileNav;
```

Comment:
Please delete this file. The useNav hook should be the only hook for the nav. There is no need to make another hook only for the mobile nav.

Also, could you add the contact us object to the useNav hook please?